docs/reference/2-ods-api/how-to-guides/how-to-create-a-custom-populated-template.md

```diff
@@ -276,9 +235,9 @@ folder and execute `nuget pack`.
 #### NuGet CLI Pack Command
 
 ```powershell
-PS C:\edfi\Ed-Fi-ODS-Implementation\DatabaseTemplate\Database> nuget pack
+PS C:\Ed-Fi-ODS-Implementation\DatabaseTemplate\Database> nuget pack
 Attempting to build package from 'Populated.Template.nuspec'.
-Successfully created package 'C:\edfi\Ed-Fi-ODS-Implementation\DatabaseTemplate\Database\Populated.Template.1.0.0.nupkg'.
+Successfully created package 'C:\Ed-Fi-ODS-Implementation\DatabaseTemplate\Database\Populated.Template.1.0.0.nupkg'.
```

Comment:
I shortened the path so that lines would not be so long. This was particularly valuable in code blocks, since code blocks are not wrapped in the HTML display - they have horizontal scrolling, which we want to avoid as much as possible.

Reply:
Good idea, adding this to our migration notes as well.